x-pack/legacy/plugins/apm/index.ts

```diff
@@ -88,22 +88,31 @@ export const apm: LegacyPluginInitializer = kibana => {
         catalogue: ['apm'],
         privileges: {
           all: {
-            api: ['apm', 'apm_write'],
+            api: ['apm', 'apm_write', 'actions-read', 'alerting-read'],
```

Comment:
`apm_write` uses snake_case. Should we stay consistent?

Reply:
Btw. Should `apm` be renamed to `apm_read`?

Reply:
ah.. `actions-read` is what's defined in the Action app endpoints...
My mind is a little hazy on this mapping concept. Are we mapping tags to privileges here? How does it work?

Reply:
I understood it last time I looked into it, but that was last year. It seems this mapping concept is not immediately intuitive so adding some comments (albeit a bit verbose) might be a good idea.

Reply:
Would just linking to `x-pack/plugins/features/common/feature_kibana_privileges.ts` where it's explained be OK? Rather than copying documentation between files?